enginetest/queries/queries.go

```diff
@@ -4456,6 +4456,18 @@ Select * from (
 		Query:    "SELECT id FROM typestable WHERE da >= subdate('2020-01-01', 1)",
 		Expected: []sql.Row{{int64(1)}},
 	},
+	{
+		Query:    "SELECT adddate(da, i32) from typestable;",
+		Expected: []sql.Row{{time.Date(2020, time.January, 4, 0, 0, 0, 0, time.UTC)}},
+	},
+	{
+		Query:    "SELECT adddate(da, concat(u32)) from typestable;",
+		Expected: []sql.Row{{time.Date(2020, time.January, 8, 0, 0, 0, 0, time.UTC)}},
+	},
+	{
+		Query:    "SELECT adddate(da, f32/10) from typestable;",
+		Expected: []sql.Row{{time.Date(2020, time.January, 1, 0, 0, 0, 0, time.UTC)}},
+	},
```

Comment:
Would be nice to have one test using `adddate` with an `INTERVAL` to exercise that code path